src/content/docs/running-kessel/monitoring-kessel/monitoring-data-replication-inventory-api.mdx

```diff
@@ -0,0 +1,70 @@
+
+Inventory API leverages Kafka Connect related metrics to understand the state of our Debezium connector and monitor its health.
+
+### Kafka Lag Exporter
+
+The [Kafka Lag Exporter](https://docs.redhat.com/en/documentation/red_hat_streams_for_apache_kafka/2.9/html/deploying_and_managing_streams_for_apache_kafka_on_openshift/assembly-metrics-str#con-metrics-kafka-exporter-lag-str) extracts additional metrics data from Kafka brokers related to offsets, consumer groups, consumer lag, and topics. It can also be configured through Streams for Apache Kafka and offers supplmental data similar to the librdkafka metrics.
+
+Inventory API leverages the Kafka Lag exporter to monitor lag between our consumer client process and message queue.
+
+### Custom Metrics
+
+While there is no shortage of ways to get metrics data from Kafka-related services, none are indicators of how your service is functioning with regards to publishing and processing events to ensure consistency and replication. Below are the custom metrics we have defined that help fill in the gaps.
+
+* **Messages Processed**: Counter that tracks events that are consumed and processed by the application and committed as completed. Used as part of capturing message processing failure rate
+* **Message Process Failures**: Counter to track whenever processing a message fails for any reason. Used as part of capturing message processing failure rate
+* **Consumer Errors**: Counter to track when the Consumer client fails for any reason. Used to calculate consumer error rates.
+* **Kafka Error Events**: Counter to track whenever a Kafka Error event is consumed for any reason. Used to calculate kafka error rates.
+* **Outbox Event Writes**: Counter to track when an event is written to the outbox table. Used for calculating end-to-end lag.
+
+## Alerting
+
+Our alerting strategy for monitoring data consistency and replication is to ensure our alerts directly align with our KPI's, plus some platform related monitoring (Database disk usage, Kubernetes metrics, etc).
+
+For Red Hat Service Provider teams implementing a similar CDC/Outbox setup, the Management Fabric Kessel team can provide some resources, templates, and tooling that may simplify some of this monitoring work for you. See our [Internal Guide](https://docs-internal-project-kessel-8b6149c673ff0ed030e64b614e2ee9fd39.pages.redhat.com/running-kessel/monitoring-kessel/monitoring-data-replication-inventory-api) for more info (requires Red Hat VPN to access).
```

Comment:
I think the static URL for this is https://project-kessel.pages.redhat.com/docs-internal